src/renderer/Generics/redesign/Splash/splash.css.ts

```diff
@@ -1,11 +1,26 @@
 import { style } from '@vanilla-extract/css';
 import { vars } from '../theme.css';
+import welcome from '../../../../../assets/icon.png';
 
 export const container = style({
+  backgroundImage: `url(${welcome})`,
```

Comment:
I don't think we need another image? I see just the top icon in the designs

Reply:
I'm seeing a splash image at the bottom here: https://www.figma.com/file/55uOYyGkAVEpvuSgYFcum3/NiceNode-Desktop-App?node-id=1115%3A30671

But.. I must've undid my change and committed this, because the image at the top of the CSS is referring to icon.png instead of artwork/welcome.png. I can fix this, and also update the top icon to the new one 👍

Reply:
Ok I see it. My b.